server/parser/src/main/java/org/eclipse/lsp/cobol/rules/procedure/statements/SearchRule.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2024 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ */
+package org.eclipse.lsp.cobol.rules.procedure.statements;
+
+import org.eclipse.lsp.cobol.cst.procedure.Statement;
+import org.eclipse.lsp.cobol.parser.hw.ParsingContext;
+import org.eclipse.lsp.cobol.rules.CobolLanguage;
+import org.eclipse.lsp.cobol.rules.LanguageRule;
+import org.eclipse.lsp.cobol.rules.procedure.ConditionExpressionRule;
+import org.eclipse.lsp.cobol.rules.procedure.IdentifierRule;
+import org.eclipse.lsp.cobol.rules.procedure.SentenceRule;
+
+/** p. 434 */
```

Comment:
is this a page reference to lang guide ?

Reply:
Yes :) I know that it's not the best way to reference, but it's better than nothing